npm-shrinkwrap.json

```diff
@@ -3728,9 +3646,9 @@
       "integrity": "sha512-zg7Hz2k5lI8kb7U32998pRRFin7zJlkfezGJjUc2heaD4Pw2wObakCDVzkKztTm/Ln7eiVvYsjqak0Ed4LkMDA=="
     },
     "aws4-axios": {
-      "version": "2.4.6",
-      "resolved": "https://registry.npmjs.org/aws4-axios/-/aws4-axios-2.4.6.tgz",
-      "integrity": "sha512-MipHf2VZYwl5DVTKUf9Be+3e20ukCknQbHVhVw+ann2UJ0KKa7XgZtAPnBjnlvrlNvomB1AYOQX0+5I/XqwPEg==",
+      "version": "2.3.1",
```

Comment:
This `aws4-axios` downgrade is a case in point re my other comment (several downgraded modules). Why does `npm` think that it needs to be downgraded to v2.3.1? It is a dependency of `etcher-sdk@6.2.1`, which is not touched by this PR as far as I can tell.